compiler/luci/pass/src/ResolveCustomOpMaxPoolWithArgmaxPass.cpp

```diff
@@ -446,6 +446,43 @@ luci::CircleNode *window_flattened_coord(const std::string &name, luci::Padding
   return argmax_cast;
 }
 
+// Creates dummy operation after floor to force circle-quantizer requantize tensor with scale << 1
+// Dealing with values of extremely different scales
+// in following binary operations hurts backend precision.
+luci::CircleNode *create_post_floor_requantize_node(luci::CircleFloor *floor)
+{
+  auto graph = floor->graph();
+  auto const origin = luci::get_origin(floor);
+  auto name = floor->name();
+
+  auto requantizer = none_act_func(graph->nodes()->create<luci::CircleDepthwiseConv2D>());
```

Comment:
`depthwise conv` was chosen in favor of other operations to prevent it's optimization (elimination of fuse)

Reply:
Maybe adding a note about why `DepthwiseConv2D` acts as dequantizer.

Reply:
Is [this](https://github.com/Samsung/ONE/pull/7516/files#diff-12b562f1cad3615f227db91860a50b75bd907dd91a2411832c79846a42213937R449-R451) comment sufficient?

Reply:
> Is this comment sufficient?

> // Creates dummy operation after floor to force circle-quantizer requantize tensor with scale << 1
// Dealing with values of extremely different scales
// in following binary operations hurts backend precision.

Explanation itself looks good. But I could not catch this `dummy` is the `CircleDepthwiseConv2D`.
I think explicitly adding a note would be nice.